Extension/src/common.ts

```diff
@@ -620,4 +620,37 @@ export function downloadFileToDestination(urlStr: string, destinationPath: strin
         request.on('error', (error) => { reject(); });
         request.end();
     });
+}
+
+export function downloadFileToStr(urlStr: string, headers?: OutgoingHttpHeaders): Promise<any> {
```

Comment:
nifty.  Should we update the query for the release JSON to use this new method too?

Reply:
Totally. I figured it was best to ask first, then I ended up forgetting :laughing: